packages/flutter/lib/src/material/radio.dart

```diff
@@ -106,6 +106,7 @@ class Radio<T> extends StatefulWidget {
     this.autofocus = false,
     this.enabled,
     this.groupRegistry,
+    this.backgroundColor,
```

Comment:
We should also add this to RadioListTile

Reply:
Similar to https://github.com/flutter/flutter/pull/169415#discussion_r2127868086, I was considering adding it with another PR. Would that be fine with you?

Reply:
yes